defold-spine/src/comp_spine_model.cpp

```diff
@@ -854,7 +905,7 @@ namespace dmSpine
                 continue;
             const Vector4 trans = component.m_World.getCol(3);
             write_ptr->m_WorldPosition = Point3(trans.getX(), trans.getY(), trans.getZ());
-            write_ptr->m_UserData = (uintptr_t) &component;
+            write_ptr->m_UserData = (uintptr_t) i;
```

Comment:
Use an integer to the component instead of a direct pointer

Reply:
I would do this too. However, see my previous notice about which index to use.